app/javascript/components/App.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { BrowserRouter, Routes, Route, Link } from "react-router-dom";
+import LoadingBar from "react-redux-loading-bar";
+import Greeting from "./Greeting";
+import store from "../redux/configureStore";
+import { Provider } from "react-redux";
+
+const App = () => (
+  <Provider store={store}>
+    <BrowserRouter>
+      <LoadingBar />
+      <Routes>
+        <Route path="/greeting" element={<Greeting />} />
+        <Route
+          path="/"
+          element={<Link to="/greeting">Get random greeting</Link>}
```

Comment:
- [ ]  It will be much better to use `components` instead of `elements` when creating your routes. It will make your code much easier to understand

Reply:
I'm using the latest react-router-dom library.

Reply:
This reply is understandable. Issue closed. :+1: